op-node/rollup/driver/sequencer.go

```diff
@@ -85,6 +88,12 @@ func (d *Sequencer) StartBuildingBlock(ctx context.Context) error {
 		return err
 	}
 
+	// Request coordinator for the permission to start building a new block.
```

Comment:
Is it more appropriate to put it at the beginning of the method

Reply:
nice catch